config/locales/en.yml

```diff
@@ -2938,10 +2938,10 @@ en:
           nth_dependent_name: Your %{ordinal} dependent's name
           spouse_name: Your spouse's name
           taxes_owed_html: |
-            <p>
-              <strong>Your refund or taxes owed</strong> <br/>
-              $0
-            </p>
+            <h2 class="text--body text--bold spacing-below-5">
```

Comment:
[pebble] per our discussion, we should move this h2 tag to the HTML and not the YML, right?

Reply:
+1 since this line has class styling in the tag